locales/en-US/server.ftl

```diff
@@ -100,6 +100,7 @@ shotPageShareButton =
     .title = Share
 shotPageCopyButton =
     .title = Copy image to clipboard
+shotPageCopyText = Copy
```

Comment:
English is unclear (noun or verb?), I'd prefer a different ID to avoid confusion, e.g. shotPageCopyActionLabel (since `shotPageCopyButton` is already in use)

Reply:
It's a verb, will update with suggested ID

Reply:
Done